packages/search-service/src/search-mapping/schema.ts

```diff
@@ -0,0 +1,31 @@
+import { Document, Model, model, Schema } from 'mongoose';
+
+export const SearchMapSchema: Schema = new Schema({
+  name: String,
```

Comment:
Add another field for SPA. That should store the reference to the SPA id from the Apps/Home Service.
```suggestion
  name: String,
  spa: { type: String, },
```